smtp/smtpc.c

```diff
@@ -243,7 +243,9 @@ smtpc_execute(struct smtpc_request *req, double timeout)
 		req->status = (int) longval;
 		req->reason = "Ok";
 		break;
+#if LIBCURL_VERSION_NUM < 0x073e00
 	case CURLE_SSL_CACERT:
```

Comment:
Nit: I would add `/* deprecated in libcurl 7.62.0 */` here as in https://github.com/tarantool/tarantool/commit/02da15f7109b8bb7921eb97a751607b9dd2885b2